tests/preview_file/custom_dir_preview.fish

```
@@ -1,4 +1,5 @@
-set fzf_preview_dir_cmd exa
-mock exa . "echo file.txt"
-set actual (__fzf_preview_file .)
+set fzf_preview_dir_cmd ls
+set dir (mktemp -d)
+touch $dir/{file.txt,.hidden}
```

Why did you have to change this test?

---

We are using command instead of eval now, but `mock` only creates functions, so the test failed: https://github.com/PatrickF1/fzf.fish/commit/e24052f1b56db82658bf78a82fb7c96f5f78f28e

---

gotcha, thanks for explaining!